src/pages/AdminManageAccess.js

```diff
@@ -86,9 +86,11 @@ class AdminManageAccess extends Component {
 
   openDialogCreate(dataRequestId, frontEndId) {
     this.setState({ showDialogCreate: true, dataRequestId: dataRequestId, frontEndId: frontEndId });
-  };
+  }
 
-  dialogHandlerCancel = async (dataRequestId, electionId) => {
```

Comment:
functional change here lines 91-93. the rest of the changes are formatting to address compilation warnings